core/include/usModuleSettings.h

```diff
@@ -43,11 +45,15 @@ US_BEGIN_NAMESPACE
  * - \e US_AUTOLOAD_PATHS A ':' (Unix) or ';' (Windows) separated list of paths
  *   from which modules should be auto-loaded.
  *
+ * \deprecated Use FrameworkFactory::newFramework(std::map<std::string, std::string> configuration) to configure the framework.
```

Comment:
Because the project is not officially affiliated with the OSGi consortium, I would like to avoid any troubles regarding the re-use of the term "OSGi". Maybe it is not an issue for string literals - I didn't consult a lawyer. But I know one project which had to change its name because the OSGi guys knocked on its doors. Granted, it was the project name itself but still...